src/lib/datepicker/datepicker.md

```diff
@@ -116,13 +116,13 @@ three pieces via injection:
  3. The message strings used in the datepicker's UI.
 
 #### Setting the locale code
-By default the datepicker will use the locale code from the `LOCALE_ID` injection token from
-`@angular/core`. If you want to override it, you can provide a new value for the token:
-
+By default `MAT_DATE_LOCALE` injection token will use the existing `LOCALE_ID` locale code from
```

Comment:
Nit: By default `MAT_DATE_LOCALE` injection token -> By default, the `MAT_DATE_LOCALE` injection token